daemon/daemon_unix.go

```diff
@@ -1560,12 +1560,15 @@ func (daemon *Daemon) statsV2(s *types.StatsJSON, stats *statsV2.Metrics) (*type
 			Usage: stats.Memory.Usage,
 			// MaxUsage is not supported
 			Limit: stats.Memory.UsageLimit,
-			// TODO: Failcnt
 		}
 		// if the container does not set memory limit, use the machineMemory
 		if s.MemoryStats.Limit > daemon.machineMemory && daemon.machineMemory > 0 {
 			s.MemoryStats.Limit = daemon.machineMemory
 		}
+		if stats.MemoryEvents != nil {
+			s.MemoryStats.Failcnt = stats.MemoryEvents.Oom
```

Comment:
Do you think this requires a comment to explain? It may not be obvious how these relate

Reply:
updated